database/json_operations.py

```diff
@@ -1,20 +1,13 @@
 import json
 import aiosqlite
+from config.storage_config import STORAGE_CONFIG
 
 
-def read_data_from_json_file(file_path):
-    print('11')
-    with open(file_path, 'r', encoding='utf-8') as file:
-        data = json.load(file)
-    return data
-
-
-async def insert_questions_data(data, db_name):
-    print('22')
-    async with aiosqlite.connect(db_name) as db:
+async def insert_questions_data(data):
```

Comment:
also this function shoul be in questions_table.py, because actions in it relate directly to questions_table